tests/aws/services/apigateway/test_apigateway_extended.snapshot.json

```diff
@@ -650,6 +651,7 @@
         },
         "id": "<api-id:1>",
         "name": "Simple PetStore (Swagger)",
+        "rootResourceId": "m7b1ab5lr9",
```

Comment:
nit: could you add a transformer for this value, so that it doesn't change every time we run test? We will still need to have it in the `skip_snapshot_verify` list because we don't support it yet, but at least it wouldn't change when regenerating.
`snapshot.add_transformer(snapshot.transform.key_value("rootResourceId"))`
I think it will be needed in both tests. Thanks a lot!